docs/config-files/style-guide.md

```diff
@@ -261,7 +261,7 @@ Whenever a parameter only allows two options, formulate the label and/or descrip
 
 A parameter must define the range of min/max values, however, that range should only be as large as is necessary. Frequently, imported config files have a range of 0-255 when the only possible range is 0-1. Please check the manual.
 
-An exception are parameters which accept a single special value outside the normal range, e.g. `10-255` and `0 (disable)`. There isn't a simple way to represent these gaps, so the value range should be `0-255` in this case.
+An exception is parameters that accept a single special value outside the normal range, e.g. `10-255` and `0 (disable)`. There isn't a simple way to represent these gaps, so the value range should be `0-255` in that case.
```

Comment:
The "is" seems weird, but then I'm not a native speaker...

Reply:
I think that's good as it's referred to the exception not the parameters but yeah seems werid to read

Reply:
Parameters that (...) are an exception.
😅 idk

Reply:
It's correct. Parameters is plural but there is only one exception.

Reply:
Like I said, I'm not a native speaker :)